docker-compose.yml

```diff
@@ -33,27 +34,27 @@ services:
 
   # https://hub.docker.com/r/localstack/localstack/
   # localstack running a fake S3
-  localstack-s3:
-    image: localstack/localstack:0.10.8
+  localstack:
+    image: localstack/localstack:0.12.2
     environment:
-      - SERVICES=s3:5000
-      - DEFAULT_REGION=us-west-2
-      - HOSTNAME=localstack-s3
-      - HOSTNAME_EXTERNAL=localstack-s3
+      - SERVICES=s3
+      - DEFAULT_REGION=us-east-1
+      - HOSTNAME=localstack
+      - HOSTNAME_EXTERNAL=localstack
     expose:
-      - 5000
+      - 4566
     ports:
-      - "5000:5000"
+      - "4566:4566"
 
-  # https://hub.docker.com/r/mozilla/antenna/
+  # https://hub.docker.com/r/mozilla/socorro_collector/
   # socorro collector collecting to a separate S3 bucket
   antenna:
     image: mozilla/socorro_collector:latest
     env_file: docker/antenna.env
     depends_on:
-      - localstack-s3
+      - localstack
     expose:
       - 8000
     ports:
       - "8888:8000"
-    command: ./bin/run_web.sh
+    command: web
```

Comment:
I'm not sure if the older version didn't set an entrypoint or what, but this runs the web command in the entrypoint in the service now.